.circleci/config.yml

```diff
@@ -866,6 +866,29 @@ jobs:
           command: yarn type:doc
       - store_artifacts:
           path: ./docs/out
+  
+  # Node upstream tests
+
+  node-upstream-node:
```

Comment:
Can this be moved to a new `upstream` workflow? Otherwise it will block code coverage and the `build` workflow from passing.

Reply:
The tests have been moved to a new workflow